tests/test_pipelines_common.py

```diff
@@ -579,6 +579,25 @@ def test_num_images_per_prompt(self):
 
                 assert images.shape[0] == batch_size * num_images_per_prompt
 
+    def test_latent_vae_decode(self):
```

Comment:
I have added in a test here for making sure that the outputs match for both latent and numpy images